src/test/java/com/shopify/stepDefinitions/AddToCartSteps.java

```diff
@@ -0,0 +1,93 @@
+package com.shopify.stepDefinitions;
+
+import com.shopify.pageObjects.CartPage;
+import com.shopify.pageObjects.CheckOutPage;
+import com.shopify.pageObjects.HomePage;
+import com.shopify.pageObjects.LogInPage;
+import com.shopify.utilityClasses.BrowserUtil;
+import com.shopify.utilityClasses.ConfigReader;
+import io.cucumber.java.en.*;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+public class AddToCartSteps {
+
+    WebDriver driver;
+    LogInPage lp;
+    HomePage hp;
+    CartPage cp;
+    CheckOutPage chkpg;
+    String expectedAmount = "$29.99";
+    String orderConfMsg ="Your order has been dispatched, and will arrive just as fast as the pony can get there!";
+
+    @Given("User launches the browser")
+    public void user_launches_the_browser() {
+        driver = BrowserUtil.getDriver(ConfigReader.get("BROWSER"));
+        lp= new LogInPage(driver);
+        hp= new HomePage(driver);
+        cp= new CartPage(driver);
+        chkpg= new CheckOutPage(driver);
```

Comment:
This pages should not be initialised at begining they should be initialised when needed

Reply:
Shifted the declaration before where they are used